src/pykeen/ablation/__init__.py

```diff
@@ -1,10 +1,270 @@
 # -*- coding: utf-8 -*-
 
-"""Ablation studies in PyKEEN."""
+"""Ablation studies in PyKEEN.
 
-from .ablation import prepare_ablation, prepare_ablation_from_config
+  "If :func:`pykeen.pipeline.pipeline` were making a maze, :func:`pykeen.hpo.hpo_pipeline` were getting
+  killed by your dead wife, then :func:`pykeen.ablation.ablation_pipeline` is washing up on
+  a beach." -Christopher Nolan
+
+An ablation study is an experiment in which components of a machine learning system are removed/replaced in order
+to measure the impact of these components on the system's performance.
+In PyKEEN, a user could for instance measure the impact of explicitly modeling inverse relations on the model's
+performance.
+
+An ablation study is defined based on a dictionary containing three sub-dictionaries: 1.) metadata,
+2.) ablation, and 3.) optuna, which we will define below.
+In the following, we have to define in several cases the dictionaries 'kwargs' and 'kwargs_ranges.
+'kwargs' defines for a specific component (e.g., interaction model or loss function) single hyper-parameter values
+such as a fixed embedding dimension of 50, whereas 'kwargs_ranges' define ranges of values.
+Note that we always have to define both dictionaries, and in cases where do not have entries for
+'kwargs' or 'kwargs_ranges', we define empty dictionaries.
+
+>>> configuration = {}
+
+Define Metadata Dictionary
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+>>> metadata = dict(
+...    title="HPO over MyData"
+... )
+
+>>> configuration['metadata'] = metadata
+
+Define Ablation Dictionary
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+>>> ablation = {}
+
```

Comment:
the way you wrote this is really counterintuitive. Each code block should be completely executable. use the 

```
.. code-block:: python

   ...
```

to make it easier to put more code together and use python commenting